server/remote/coding/coding.go

```diff
@@ -172,6 +172,7 @@ func (c *Coding) Repo(ctx context.Context, u *model.User, owner, name string) (*
 		return nil, err
 	}
 	return &model.Repo{
+		// TODO RemoteID:     project.ID,
```

Comment:
```suggestion
		// TODO(1138) RemoteID:     project.ID,
```

and add new headline at #1138 and describe the issue ...